marimo/_utils/requests.py

```diff
@@ -0,0 +1,144 @@
+import http.client
```

Comment:
this might be useful over time with the other fetching we do without `requests` or `urlib3`